### PR TITLE
phpcs: add page

### PR DESCRIPTION
**Pull request by pxgamer**
----

## Basic Check

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

## References

Official Page: https://github.com/squizlabs/PHP_CodeSniffer

### CODE REVIEW

File: pages/common/phpcs.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- Display a list of installed coding standards:
+
+`phpcs -i`
+
+- Specify a coding standard to validate against:
+
+`phpcs {{path/to/directory}} --standard {{standard}}`
+
+- Specify file extension(s) to include when sniffing:
```
Comment (sbrl) [inline pages/common/phpcs.md]: Is there a way to exclude files too?

Comment (pxgamer) [reply pages/common/phpcs.md]: There's an `--ignore` option which can be used.

**Comment by pxgamer** [comment]
@jsonbruce, I thought I'd try it out. At least so we can see how it looks. :+1:

File: pages/common/phpcs.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# phpcs
+
+> Tokenize PHP, JavaScript and CSS files to detect violations of a defined set of coding standards.
+
+- Sniff the specified directory for issues:
```
Comment (agnivade) [inline pages/common/phpcs.md]: What is the default coding standard it checks against ?

Comment (pxgamer) [reply pages/common/phpcs.md]: It defaults to the PEAR standard.

Comment (agnivade) [reply pages/common/phpcs.md]: Ok, perhaps we can specify that in brackets to make it extra clear. It was a natural question that came to my mind.

Comment (pxgamer) [reply pages/common/phpcs.md]: Makes sense. :smiley:

File: pages/common/phpcs.md (hunk 1 above)
Comment (sbrl) [reply pages/common/phpcs.md]: Perhaps we could one of the lesser used options out for that, or include it in this example? It sounds useful to me. Thoughts?

Comment (pxgamer) [reply pages/common/phpcs.md]: I guess maybe `--ignore` instead of `-l` (no recursion)? 🤔

Comment (sbrl) [reply pages/common/phpcs.md]: Yeah, I guess. I guess you can always use shell globs with it (i.e. `badfolder/**` will grab _everything_ in `badfolder/`)

**Pull request merged**